internal/scm/github/util.go

```diff
@@ -44,3 +44,17 @@ func chunkSlice[T any](stack []T, chunkSize int) [][]T {
 
 	return append(chunks, stack)
 }
+
+// mapValues returns a new array containing all the values of the supplied map,
+// in iteration (i.e. non-deterministic) order.
+func mapValues[K comparable, V any](source map[K]V) []V {
```

Comment:
To bad this is (still?) only in the experimental maps package 
https://pkg.go.dev/golang.org/x/exp/maps#Values

Reply:
I think this is slated for inclusion in 1.22? Happy to shift across to the `x` package, IIRC the API of this pretty close to what will end up being accepted.

What's your policy on adding the dependency?

Reply:
I'll add a commit that adds the dependency, it can be dropped if you don't want it.

Reply:
I think it's okay to add this dependency if it is likely that it will added to the stdlib soon 😄

Reply:
I'm currently on holidays, I should be able to look at this in the next day or two

Reply:
Removed